docs/versioned/spark/spark-structured-streaming.md

```diff
@@ -44,7 +44,7 @@ val df = spark.readStream
 ```
 
 {{< hint warning >}}
-Iceberg only supports reading data from append snapshots. Overwrite snapshots cannot be processed and will cause an exception. Similarly, delete snapshots will cause an exception by default, but deletes may be ignored by setting `streaming-skip-delete-snapshots=true`.
+Iceberg only supports reading data from snapshots of type `Append` or `Replace`. The snapshots of type `Delete` and `Overwrite` cannot be processed and will cause an exception. The spark options `streaming-skip-delete-snapshots` and `streaming-skip-overwrite-snapshots` may be set to `true` to skip snapshots of type `Delete` and `Overwrite`, respectively, in cases where it's acceptable to ignore the table changes from those snapshots.
```

Comment:
I think this should have fewer changes. It looks like this is just adding the `streaming-skip-overwrite-snapshots` property and I think it should do that using the same wording that was originally there. There isn't a need to change the sentence about delete snapshots.

Also, if you want to mention replace snapshots, we'll have to do some additional work. We don't want to assume that the reader is familiar with Iceberg internals or the spec, which is why we replace out originally. If you want to mention them, I think we should refer to them as "data rewrites that do not change table data".

Reply:
That's a fair point about `Replace` snapshots. Even as somebody working on Iceberg, I wasn't able to point out what was really meant by that until looking into it a bit more. It's definitely an implementation detail.

Reply:
addressed the comments @rdblue @kbendick 
pl. take another look.